code/game/machinery/suit_storage_unit.dm

```diff
@@ -649,18 +663,12 @@
 	if(!panel_open)
 		return
 	else
-		if(uv_super)
-			to_chat(user, "<span class='notice'>You slide the dial back towards \"185nm\".</span>")
-			uv_super = FALSE
-		else
-			to_chat(user, "<span class='notice'>You crank the dial all the way up to \"15nm\".</span>")
-			uv_super = TRUE
+		uv_super = !uv_super
 
 /obj/machinery/suit_storage_unit/proc/togglesafeties(mob/user)
```

Comment:
```suggestion
/obj/machinery/suit_storage_unit/proc/togglesafeties()
```
No longer uses an user param so it can be removed. Ensure to change the usages as well